src/services/waku.ts

```diff
@@ -0,0 +1,140 @@
+import { v4 as uuid } from "uuid";
+import {
+  PUBSUB_TOPIC,
+} from "@/constants";
+import { http } from "@/utils/http";
+
+export type Message = {
+  payload: string,
+  contentTopic: string,
+  version: number,
+  timestamp: number
+};
+
+type EventListener = (event: CustomEvent) => void;
+
+const SECOND = 1000;
+const LOCAL_NODE = "http://127.0.0.1:8645/";
+const FILTER_URL = "/filter/v2/";
```

Comment:
same as other comment. we assume that the underlying node runs relay, so no need to rely on light protocols.
see for reference: https://waku-org.github.io/waku-rest-api/